src/blog/2022/009-es2018.md

```diff
@@ -0,0 +1,58 @@
+---
+
+title: Core Libraries Migrated To ES2018
```

Comment:
I'd simplify this to `Migration to ES2018`. Not sure if it is clear what the "core libraries" are.